studioShared/schemas/fields/customerCaseProjectInfo.ts

```diff
@@ -0,0 +1,41 @@
+import { defineField } from "sanity";
+
+export const customerCaseProjectInfo = defineField({
+  name: "customerCaseProjectInfo",
+  title: "Project Info",
+  description: "General information about the project",
+  type: "object",
+  fields: [
+    {
+      name: "customer",
+      description: "The customer for the project",
+      type: "internationalizedArrayString",
+    },
+    {
+      name: "projectName",
+      description: "The name of the project",
+      type: "internationalizedArrayString",
+    },
+    {
+      name: "projectDuration",
+      description: "The duration of the project",
+      type: "internationalizedArrayString",
+    },
+    {
+      name: "sector",
+      description: "The sector of the project",
+      type: "internationalizedArrayString",
+    },
+    {
+      name: "delivery",
+      description: "The delivery of the project",
+      type: "internationalizedArrayString",
+    },
+    {
+      //TODO: We should be able to select the consultants from a list
+      name: "consultants",
+      description: "The consultants for the project",
+      type: "internationalizedArrayString",
```

Comment:
Don't need to translate the consultants right?

Reply:
Hehe no we dont!